src/accounts/AccountPage.tsx

```diff
@@ -68,18 +68,18 @@ const AccountAboutPage: FC = () => {
     <AccountTabContainer activeTab="about">
       <>
         {userAccounts && userAccounts.length >= 2 && (
-          <React.Fragment>
+          <div>
             <Button
               text="Switch Account"
               icon={IconFont.Switch_New}
               onClick={showSwitchAccountDialog}
               testID="user-account-switch-btn"
             />
-            <hr />
-          </React.Fragment>
+            <hr style={{marginTop: '32px'}} />
```

Comment:
can you move this up to lines 64/65 like the other style declarations to be consistent with this file? Anonymous style declarations like this cause re-renders.

Reply:
For sure!